site/content/docs/browser-checks/playwright-test.md

```diff
@@ -135,22 +85,27 @@ Playwright Test Runner offers hook functions such as `test.afterEach` and `test.
 You can find more information on available methods in the [official documentation](https://playwright.dev/docs/api/class-test).
 
 ### Viewing trace files
-When a `@playwright/test` test case fails, Checkly will record and make its trace files available via the UI. 
-This makes it very easy to inspect individual traces and debug failing tests.
+When a `@playwright/test` test case fails, Checkly will record and make its trace files available via the UI. You can download the trace files for manual inspection or view them directly with [trace.playwright.dev](https://trace.playwright.dev). 
 
-You can download the trace files for manual inspection or view them directly with [trace.playwright.dev](https://trace.playwright.dev).
+Using the Playwright Trace Viewer you can effortlessly view your test, skip back and forth between actions, view snapshots and metadata, and more. This makes it very easy to inspect individual traces and debug failing tests.
 
 ![checkly-pw-traces](/docs/images/browser-checks/pwt_traces.gif)
 
 > **Note:** When running tests from the editor page, trace files are always available for download and preview, 
 > regardless of whether the check is passing or failing. For scheduled check runs traces are only preserved when the check failed.
 
 ### Video recordings
-When a `@playwright/test` test case fails, Checkly will record a video for each page navigation and make it available in the UI.
+When a `@playwright/test` test case fails, Checkly will record a video for each page navigation and make it available in the UI. It is a great tool to get a first look of the actions and their outcome to quickly identify what failed, and to visualize regressions.
 
 Here's an example of a Playwright Test script that fails, and provides a video of the test sequence.
 
 ![checkly-pw-videos](/docs/images/browser-checks/pwt_videos.gif)
 
 > **Note:** When running tests from the editor page, video files are always available for download and preview, 
 > regardless of whether the check is passing or failing. For scheduled check runs videos are only preserved when the check failed.
+
+## Overhead issue
+
+A check using Playwright Test Runner will currently run around 30-50% longer than a regular Playwright check. This is caused by the additional assets of traces and videos. We are aware of this and are investigating solutions.
```

Comment:
Someone may think we are "hiding" this somewhat crucial information at the bottom of the page. Let's make the first sentence **bold** to make it more visible

Reply:
@jan-osch I was also thinking of where to put this. The hierarchy is currently like below, but we could also just put all sections on the same level and place the `overhead` section as the second in order. I just didn't think it would count under `features`, so it landed in the bottom.

<img width="276" alt="Screenshot 2023-02-10 at 14 40 35" src="https://user-images.githubusercontent.com/54396648/218119151-efc4c84f-597a-4d76-8d78-507ad2ee4ab3.png">

Reply:
nevermind, I moved this to the top as a warning